src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs

```diff
@@ -66,6 +68,7 @@ public async Task<IActionResult> AddDraftapprenticeships(BulkUploadAddDraftAppre
                 new BulkUploadAddDraftApprenticeshipsCommand
                 {
                     ProviderId = request.ProviderId,
+                    FileUploadLogId = request.FileUploadLogId,
```

Comment:
or a test for this

Reply:
Added param in existing test WhenAddingBulkUploadDraftApprenticeships